tests/unit/coverage-setup.cjs

```diff
@@ -0,0 +1,26 @@
+const { createInstrumenter } = require('istanbul-lib-instrument');
+const { resolve } = require('path');
+const { readFileSync } = require('fs');
```

Comment:
@THEBOSS0369  We're not using CommonJS modules. You should only use the ES Module `import` statements.